deploy/role_binding.yaml

```diff
@@ -110,3 +110,17 @@ roleRef:
   kind: Role
   name: resultserver
   apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: prometheus-k8s
+  namespace: openshift-compliance
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: prometheus-k8s
+subjects:
+  - kind: ServiceAccount
+    name: prometheus-k8s
+    namespace: openshift-monitoring
```

Comment:
Would this work also if openshift-monitoring was not installed?

Reply:
It's possible to create a rolebinding that references a nonexistant SA, if that's what you mean

Reply:
yes, thanks